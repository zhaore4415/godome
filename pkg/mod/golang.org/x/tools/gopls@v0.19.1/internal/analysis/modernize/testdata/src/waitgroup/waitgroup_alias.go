package waitgroup

import (
	"fmt"
	sync1 "sync"
)

func _() {
	var wg sync1.WaitGroup
	wg.Add(1) // want "Goroutine creation can be simplified using WaitGroup.Go"
	go func() {
		defer wg.Done()
		fmt.Println()
	}()

	wg.Add(1) // want "Goroutine creation can be simplified using WaitGroup.Go"
	go func() {
		fmt.Println()
		wg.Done()
	}()
}
