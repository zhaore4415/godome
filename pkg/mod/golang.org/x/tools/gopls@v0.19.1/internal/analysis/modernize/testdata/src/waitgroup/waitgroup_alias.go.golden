package waitgroup

import (
	"fmt"
	sync1 "sync"
)

func _() {
	var wg sync1.WaitGroup
	// want "Goroutine creation can be simplified using WaitGroup.Go"
	wg.Go(func() {
		fmt.Println()
	})

	// want "Goroutine creation can be simplified using WaitGroup.Go"
	wg.Go(func() {
		fmt.Println()
	})
}